<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginEditor</class>
 <widget class="QWidget" name="PluginEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plugin Info</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
   <item row="0" column="0">
    <widget class="QLabel" name="label_1">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Version:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Compatibility version:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Vendor:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Url:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" rowspan="2">
    <widget class="QTextBrowser" name="textBrowser_Description">
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer_1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>License:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2" rowspan="2">
    <widget class="QTextBrowser" name="textBrowser_License">
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Dependencies:</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEdit_Name"/>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEdit_Version"/>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEdit_CompatibilityVersion"/>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="lineEdit_Vendor"/>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="lineEdit_Url"/>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="lineEdit_Copyright"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Copyright:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QListView" name="listView_Dependencies"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
